data/json/npcs/NC_DEBUG.json

```diff
@@ -0,0 +1,17 @@
+[
+  {
+    "type" : "item_group",
+    "id" : "NC_DEBUG_worn",
+    "items":[ ["towel", 100] ]
+  },
+  {
+    "type" : "item_group",
+    "id" : "NC_DEBUG_carried",
+    "items":[ ["mininuke", 100] ]
+  },
+  {
+    "type" : "item_group",
+    "id" : "NC_DEBUG_weapon",
+    "items":[ ["soap", 100] ]
```

Comment:
Soap in "weapon" group and mininuke in "carried" group? Am I too serious about some kind of joke I missed?